detail/pod/index.vue

```diff
@@ -0,0 +1,215 @@
+<script>
+import { NODE } from '@/config/types';
+import createEditView from '@/mixins/create-edit-view';
+import DetailTop from '@/components/DetailTop';
+import ResourceTabs from '@/components/form/ResourceTabs';
+import LinkDetail from '@/components/formatter/LinkDetail';
+import LiveDate from '@/components/formatter/LiveDate';
+import SortableTable from '@/components/SortableTable';
+import Networking from '@/components/form/Networking';
+import Tab from '@/components/Tabbed/Tab';
+import Scheduling from '@/components/form/Scheduling';
+import Container from '@/components/form/Container';
+import PodSecurity from '@/components/form/PodSecurity';
+
+export default {
+
+  name:       'PodDetail',
+  components: {
+    DetailTop,
+    ResourceTabs,
+    LinkDetail,
+    LiveDate,
+    SortableTable,
+    Tab,
+    Networking,
+    Scheduling,
+    Container,
+    PodSecurity
+  },
+
+  mixins: [createEditView],
+
+  data() {
+    return {
+      workload: null, node: null, nodes: []
+    };
+  },
+
+  computed:   {
+    detailTopColumns() {
+      return [
+        {
+          title:   'Workload',
+          name:  'workload'
+        },
+        {
+          title:   'Pod IP',
+          content: this.value.status.podIP
+        },
+        {
+          title: 'Node',
+          name:  'node'
+        },
+        {
+          title:   'Pod Restarts',
+          content: (this.value?.status?.containerStatuses || [])[0]?.restartCount
+        },
+        {
+          title:   'Created',
+          name:  'created'
+        }
+      ];
+    },
+
+    containerHeaders() {
+      return [
+        {
+          name:      'status',
+          label:     'Status',
+          value:     'stateDisplay',
+          formatter: 'BadgeStateFormatter'
+        },
+        {
+          name:  'name',
+          label: 'Name',
+          value: 'name'
+        },
+        {
+          name:  'image',
+          label: 'Image',
+          value: 'image'
+        },
+        {
+          name:  'imagePullPolicy',
+          label: 'Image Pull Policy',
+          value: 'imagePullPolicy'
+        }
+      ];
+    },
+
+    containers() {
+      return (this.value.spec.containers || []).map((container) => {
+        container.status = this.findContainerStatus(container);
+        if (container.status) {
+          container.stateBackground = this.value.containerStateColor(container.status);
+          container.stateDisplay = this.value.containerStateDisplay(container.status);
+        }
+
+        return container;
+      });
+    }
+  },
+
+  mounted() {
+    this.findWorkload();
+    this.findNode();
+  },
```

Comment:
These can go into `fetch()` now, then wrap the template body with

```html
<Loading v-if="$fetchState.pending" />
<div v-else>
...
</div>
```

So that an incomplete page isn't rendered and then adjusted